<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zuhexingdaikuan</class>
 <widget class="QWidget" name="zuhexingdaikuan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>组合型贷款</string>
  </property>
  <widget class="QPushButton" name="pushButton_kaishiJisuan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>440</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>190</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>元</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>310</y>
     <width>54</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金贷款额：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gongjijinDaikuane">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_huankuanFangshi">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>等额本息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>等额本金</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>还款方式：    </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_anjieNianshu">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>310</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>按揭年数：    </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>公积金利率：  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_gongjijinLilv">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>250</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>商贷利率：    </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_shangdaiLilv">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>商业贷款额：  </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_shangyeDaikuane">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string> 温馨提示：请您输入有意义的数字！</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_shangyeDaikuane</sender>
   <signal>textChanged(QString)</signal>
   <receiver>zuhexingdaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_shangdaiLilv</sender>
   <signal>textChanged(QString)</signal>
   <receiver>zuhexingdaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_gongjijinDaikuane</sender>
   <signal>textChanged(QString)</signal>
   <receiver>zuhexingdaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_gongjijinLilv</sender>
   <signal>textChanged(QString)</signal>
   <receiver>zuhexingdaikuan</receiver>
   <slot>on_text_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_text_changed()</slot>
 </slots>
</ui>
